pcsx2/gui/FrameForGS.cpp

```diff
@@ -173,6 +184,8 @@ void GSPanel::DoResize()
 		zoom = std::max( (float)arr, (float)(1.0/arr) );
 
 	viewport.Scale(zoom, zoom*g_Conf->GSWindow.StretchY.ToFloat()/100.0 );
+	if (viewport.x == (double)client.GetWidth() && viewport.y == (double)client.GetHeight() && EmuConfig.Gamefixes.FMVinSoftwareHack && g_Conf->GSWindow.IsFullscreen)
```

Comment:
Sorry, I wasn't clear. You could replace
`viewport.x == (double)client.GetWidth() && viewport.y == (double)client.GetHeight()`
with
`viewport == client`